src/test/obj_pmalloc_mt/obj_pmalloc_mt.c

```diff
@@ -155,6 +155,39 @@ alloc_free_worker(void *arg)
 	return NULL;
 }
 
+#define OPS_PER_TX 10
+#define TX_PER_TH 100
+#define STEP 8
+#define TEST_LANES 4
+
+static void *
+tx2_worker(void *arg)
+{
+	struct worker_args *a = arg;
+
+	for (int i = 0; i < TX_PER_TH; ++i) {
+		PMEMoid oids[OPS_PER_TX];
+		TX_BEGIN(a->pop) {
+			for (int i = 0; i < OPS_PER_TX; ++i) {
+				oids[i] = pmemobj_tx_alloc(ALLOC_SIZE, a->idx);
+				for (int j = 0; j < ALLOC_SIZE; j += STEP) {
+					pmemobj_tx_add_range(oids[i], j, STEP);
```

Comment:
For me? Thanks!
I think
1) either the documentation should state that the real size is always a multiple of 8
2) or the tests should not rely on it at all